content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -40,6 +45,14 @@ etcd-peer-client-cert-auth: "true"
 etcd-peer-trusted-ca-file: "/path/to/your/peer/ca/file"
 {{< /code >}}
 
+To properly secure etcd communication, replace the default parameter values in your backend store configuration with non-default versions of these certificates, keys, and URLs:
+
+ - A certificate and key for the `etcd-cert-file` and `etcd-key-file` to secure client communication
+ - A certificate and key for the `etcd-peer-cert-file` and `etcd-peer-key-file` to secure cluster communication
+ - Non-default values for `etcd-listen-client-urls`, `etcd-listen-peer-urls`, and `etcd-initial-advertise-client-urls`
+
```

Comment:
The [issue notes](https://github.com/sensu/sensu-docs/issues/2989) don't mention `etcd-trusted-ca-file` or `etcd-peer-trusted-ca-file` -- do those need to be listed here too?

Reply:
These flags are often used, but not strictly required for mutual TLS. Custom CA certificates can be added to the system trust store in a way that allows mutual TLS to work without providing values for these flags.